kolibri/plugins/facility/assets/src/views/DataPage/ImportInterface/index.vue

```diff
@@ -68,31 +74,31 @@
       };
     },
     computed: {
+      ...mapGetters('manageCSV', ['exported']),
       ...mapState('manageCSV', ['exportUsersStatus', 'exportUsersFilename']),
       isExporting() {
         return this.exportUsersStatus === UsersExportStatuses.EXPORTING;
       },
     },
-    watch: {
-      exportUsersStatus(val) {
-        if (val == UsersExportStatuses.FINISHED) {
-          window.open(
-            urls['kolibri:kolibri.plugins.facility:download_csv_file'](
-              this.exportUsersFilename,
-              this.$store.getters.activeFacilityId
-            ),
-            '_blank'
-          );
-        }
-      },
-    },
     methods: {
       ...mapActions('manageCSV', ['startExportUsers']),
       exportCsv() {
         this.startExportUsers();
       },
+      downloadCsv() {
+        window.open(
+          urls['kolibri:kolibri.plugins.facility:download_csv_file'](
+            this.exportUsersFilename,
+            this.$store.getters.activeFacilityId
+          ),
+          '_blank'
+        );
+      },
     },
     $trs: {
+      generateCSV: {
+        message: 'Generate CSV file',
```

Comment:
Is it possible to implement the string change that we have for logs?
`Generate CSV file` when there's none available, and `Generate a new CSV file` when there is an older one present.